assets/src/js/script.js

```diff
@@ -469,7 +469,26 @@ jQuery(function($) {
         form_data =
           self.serialize() + '&action=dokan_settings&form_id=' + form_id;
 
-      self.find('.ajax_prev').append('<span class="dokan-loading"> </span>');
+      var isDisconnect = false;
+      var selectors = 'input[name="settings[bank][disconnect]"], input[name="settings[paypal][disconnect]"], input[name="settings[skrill][disconnect]"], input[name="settings[dokan_custom][disconnect]"]';
+      if (self.find(selectors).length > 0){
+        isDisconnect = true;
+        var nonce = self.find('input[name="_wpnonce"]').val();
+        self.find('input[type=text]').val('');
+        self.find('textarea').val('');
+        self.find('input[type=checkbox]').prop('checked', false);
+        self.find('#ac_type').prop('selectedIndex', 0);
+
+        self.find('input[name="_wpnonce"').val(nonce);
+        form_data = self.serialize() + '&action=dokan_settings&form_id=' + form_id;
```

Comment:
this line is  similar to line  number 469?

Reply:
yes, same, but the data will be different, as we have cleared the form fields to disconnect it